source/content/object-cache.md

```diff
@@ -131,9 +127,9 @@ All plans except for the Basic plan can use Redis. Sandbox site plans can enable
 
 <Tab title="Drupal 8" id="d8-install">
 
-1. Enable the Redis cache server from your Pantheon Site Dashboard by going to **Settings** > **Add Ons** > **Add**. It may take a couple minutes for the Redis server to come online.
+1. Enable Object Cache from your Pantheon Site Dashboard by going to **Settings** > **Add Ons** > **Add**. It may take a couple minutes for Object Cache to come online.
 
-1. Install and activate the [Redis](https://www.drupal.org/project/redis) module from Drupal.org.
+1. Install and activate the [Redis](https://www.drupal.org/project/object-cache) module from Drupal.org.
```

Comment:
broken link - the Drupal project is located at https://www.drupal.org/project/redis